### PR TITLE
Update nanoid

### PR DESCRIPTION
**Pull request by klappradla**
Since Gitlab is down… Some minor updates here 🧹

Changes:

- Micro formatting update to README
- Micro formatting update to linting README
- Patch [CVE-2021-23566](https://github.com/advisories/GHSA-qrpm-p2h7-hrv2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,15 @@ This is where you’ll find out How We Do Frontend Things at Bitcrowd™.
...
+## Basic frontend tools
+
 * [Homebrew](http://brew.sh/) - Used on OS X to install almost everything else.
 * [nvm (node version manager)](https://github.com/creationix/nvm) - Enables use of multiple Node.js versions.
 * [Rbenv](https://github.com/rbenv/rbenv) - Enables use of multiple versions of Ruby.
 * [npm](https://www.npmjs.com/) - Package manager for Node.js.
-* [Gulp](http://gulpjs.com/) - Javascript build and automation system.
-* [Grunt](http://gruntjs.com/) - Frontend javascript task runner.
+* [Gulp](http://gulpjs.com/) - JavaScript build and automation system.
+* [Grunt](http://gruntjs.com/) - Frontend JavaScript task runner.
```
Comment (planktonic) [inline README.md]: We can probably just remove these — they fell out of use years ago :D wdyt?

**Review by planktonic — APPROVED** [review]
dependency update magician 💚🪄

File: README.md (hunk 1 above)
Comment (klappradla) [reply README.md]: Yep. Should bitstyles also be listed here somewhere?

Comment (planktonic) [reply README.md]: Maybe, tho it doesn’t really fit with the rest of these tools — they’re more like build tools. If we’re including the frameworks etc we use, that should maybe include react, storybook etc.

**Pull request merged**